packages/server/modules/core/domain/objects/operations.ts

```diff
@@ -54,3 +54,9 @@ export type CreateObjectsBatched = (params: {
   objects: RawSpeckleObject[]
   logger?: Logger
 }) => Promise<boolean>
+
+export type CreateObjectsBatchedAndNoClosures = (params: {
```

Comment:
This interface is not needed, both functions should implement the `CreateObjectsBatched` interface, that is the point, that he no closures is a drop in replacement from the caller's perspective

Reply:
I checked, they are actually functionally different right now - one returns bool, the other returns string[]